040-SDK/101-get.mdx

```diff
@@ -825,7 +825,7 @@ You can continue like this until `"more"` is returned as `false`.
 
 ### Offset-based Pagination
 
-The offset-page pagination limit is 50,000 records. In most cases, you should use the cursor-based pagination.
+Offset-based pagination provides more flexibility and allows you to request a specific page (using `offset`) within a total amount of records given a specific page `size`. This flexibility comes at a performance cost since the amount of total records needs to be known. To prevent slow response times, Xata sets a safeguard limit of 50,0000 total records that can be paginated using the offset style.
```

Comment:
```suggestion
Offset-based pagination provides more flexibility and allows you to request a specific page (using `offset`) within a total amount of records given a specific page `size`. This flexibility comes at a performance cost since the amount of total records needs to be known and very large offset values are expensive for the database to compute. To prevent slow response times, Xata sets a safeguard limit of 50,000 total records that can be paginated using the offset style.
```